### ISSUE TITLE
Dynamic Topic Modelling (Quantized Models / Vectorized DBs)

### ISSUE THREAD

**Issue by firmai** [MaartenGr/BERTopic#1927]
It would be interesting to see if DTM ius possible with quantized models.

**Comment by MaartenGr** [comment]
Thanks for the suggestion! Just to be sure I understand correctly, did you mean using Dynamic Topic Modeling using quantized LLMs?

**Comment by firmai** [comment]
Yes exactely, I think this is a very interesting research direction. Simple but powerful idea.

**Comment by MaartenGr** [comment]
Thanks, I understand! It should be doable to implement here since it only requires calculating the representations using the representation models that are already there. Unfortunately, since I work on this in my spare time, I have limited time available to work on these kinds of features. Any help would be appreciated if anybody is interested!
